pytorch/jobs/docker/1.9/py3/Dockerfile.gpu

```diff
@@ -16,10 +16,13 @@ ENV PYTHONIOENCODING=UTF-8
 ENV LANG=C.UTF-8
 ENV LC_ALL=C.UTF-8
 
-ARG PYTHON=python3.7
+
+ARG PYTHON=python3.8
```

Comment:
Are we OK with this? Did you check with Christian?

Reply:
This was done as part of the upgrade/ testing last night.

Reply:
Python 3.8 is okay. We support it with our repositories and have tested all known use cases.